src/release/breaking-changes/boxdecoration-shape-migration.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Migration guide for BoxDecoration.circle
+description: Removal of BoxDecoration.circle and how to migrate
+---
+
+## Summary
+
+This change removes `BoxDecoration(shape: BoxShape.circle)`. Existing apps
+should use `ShapeDecoration(shape: CircleBorder())` instead.
+
+## Context
+
+BoxDecoration.shape uses an enum. Each new enum value requires new code
```

Comment:
**nit**: For consistency with [line 26](https://github.com/flutter/website/pull/7704/files#diff-0d701f8e6804cadb5e0a2f3fedd12633cf31544c7d2d29efcd6f36b94a2443bbR26), change this to `BoxDecoration.shape`.

**question:** Should this be `BoxDecoration(shape)`? If the code was `BoxDecoration{shape: BoxShape.circle}`, I could see this being `BoxDecoration.shape`, but it's not.

Reply:
The code is `BoxDecoration(shape: BoxShape.circle)`

Reply:
**question:** Right. So, why is it referred to as `BoxDecoration.shape` instead of `BoxDecoration(shape)`?

Reply:
Because I'm referring to the property value, like `A.b is an enum, so it has this bad behavior`.

If you do a = BoxDecoration(shape: BoxShape.circle), then a.shape, you get what I'm referring to.

It is not a big deal, I can make the way you want, both work.

Reply:
I tried to reword a bit, it is weird, because "`BoxDecoration(shape: BoxShape.circle)` uses an enum`". BoxDecoration.shape is the direct property and how it is used in the documentation since it is a property.

Reply:
@bernaferrari : If I'm off on this, I'll accept what you have here. No need to pretzel this.